timesketch/lib/datastores/elastic.py

```diff
@@ -179,6 +210,7 @@ def _build_query_dsl(query_dsl, timeline_ids):
                 'passed timelines are not a list.')
             return query_dsl
 
+        # pylint: disable=use-a-generator
```

Comment:
you may need to remove this linter statement, since we pinned to pylint 2.6, which doesn't `know about this`

Reply:
done.